src/ledger/conformance-test-vectors/README.md

```diff
@@ -14,12 +76,18 @@ cabal test cardano-ledger-conway
 tar czf vectors.tar.gz eras/conway/impl/dump/*
 ```
 
-Ledger V9 tests were removed, as well as two tests (BodyRefScriptsSizeTooBig,
-TxRefScriptsSizeTooBig) that produced very large sequences of transactions
-without a correspondingly significant impact on coverage.
+After that, we performed some hand curation:
+
+- Ledger V9 tests were removed, since Amaru is currently only focused on Ledger
+  v10
```

Comment:
Are v9 and v10 here the cardano protocol version? Isn't ledger versioned through eras? e.g. V10 and V11 are both Conway era and hence no difference between V10 and V11 must exist (in serialization)?